deliveries/inkathon-1-2.md

```diff
@@ -0,0 +1,56 @@
+# Milestone Delivery :mailbox:
+
+**The invoice has been send out correctly for this milestone and the delivery is according to the official [milestone delivery guidelines](https://github.com/use-inkubator/Support-Docs/blob/master/milestone-deliverables-guidelines.md).**
+
+**Application Document:** https://github.com/use-inkubator/Ecosystem-Grants/blob/master/applications/inkathon.md
+
+## Context
+
+We're happy to finally deliver a complete rewrite of the [inkathon Boilerplate](https://inkathon.xyz/) now supporting ink! v6 and much more as it includes:
+
+1. 🦑 A new **starterkit application**: https://inkathon.xyz
+2. 📚 A new **standalone documentation**: https://docs.inkathon.xyz
```

Comment:
The documentation is really slick! I also like that you chose the versioning scheme "inkathon v6", makes it less complicated for users to grasp what is compatible with what.

On the bottom of https://docs.inkathon.xyz/about you link to the changelog for inkathon v6. You could think about adding a link to https://use.ink/docs/v6/faq/migrating-from-ink-5-to-6.